Hollow-Knight-Dev/skirunmap
# docs: set up environment

[Website URL](https://skirunmap.web.app/)

## Description
- Create a vite project, with React, TypeScript.
- Install Tailwind, Prettier, ESLint, Firebase.

## Test Plan
- Check whether settings are appropriate.
- Check whether open the website can see the text "Ski Run Map".
